roles/ceph-config/templates/ceph.conf.j2

```diff
@@ -126,7 +126,7 @@ rgw frontends = {{ frontend_line(radosgw_frontend_type) }} {{ radosgw_frontend_o
 {% if 'num_threads' not in radosgw_frontend_options %}
 rgw thread pool size = {{ radosgw_thread_pool_size }}
 {% endif %}
-{% if rgw_multisite | bool %}
+{% if hostvars[host]['rgw_multisite'] | default(False) %}
```

Comment:
you still need the bool filter at the end

Reply:
@alimaredia